app/main/helpers/s3.py

```diff
@@ -1,8 +1,11 @@
 import os
 import boto
+import boto.exception
 import datetime
 import mimetypes
 
+from boto.exception import S3ResponseError  # noqa
```

Comment:
S3ResponseError is not used in the file itself, but importing it here allows me to import it from `helpers.s3` in `validation_tools.py` and tests. That means we don't have `boto` imported anywhere else and can later replace the exception with our own if we have to.

I could drop the `# noqa` since it doesn't cause issues with our build, but I think it's a good indicator that the fact that the import is not used in the file is what we actually want.